pkg/metadata/metadata.go

```diff
@@ -90,3 +92,14 @@ func (metadata Metadata) FilterNoErrorLevel() (notErrorLevel, errorLevel Metadat
 	}
 	return
 }
+
+func FilterNoErrorLevel2PB(metas []*commonpb.MetadataField) (notErrorLevel, errorLevel []*commonpb.MetadataField) {
```

Comment:
the input is already pb, so the function name should be `FilterNoErrorLevelMeta`

Reply:
adjusted